vuepress/docs/next/tutorials/create/ms/run-local.md

```diff
@@ -2,159 +2,94 @@
+```
+When the run is complete, you should see the details MFE with the ID 1 loaded.
 
+>If you want to modify a different row in the database, edit `microfrontends/conference-details/public/index.html` file. Change the `id` attribute in this line:
+``` html
+    <my-entity-details id="1" />
+```
 
 ### Keycloak Settings and Issues 
-1. Change Keycloak Dev Settings
-
-If you want to use another Keycloak installation or change the service-URL of your widget, change the parameters set in the `.env.local` file generated by the Entando Blueprint in the root folder of your widgets. For instance, `ui/widgets/YOUR-ENTITY-NAME/tableWidget/.env.local`
+1. Change Development Settings
 
-By default these variables are set to:
+* If you want to use another Keycloak installation, modify the parameter in the Keycloak YAML file located in the `svc/` folder.
+* To change the `service-url` of the widget, use the [ent bundle CLI](../../../docs/getting-started/ent-api.md) to add the new URL in the API claim. 
 
-    REACT_APP_SERVICE_URL=http://localhost:8081/services/YOUR-APPLICATION-NAME/api
-    REACT_APP_KEYCLOAK_URL=http://localhost:9080/auth
-    REACT_APP_KEYCLOAK_REALM=jhipster
-    REACT_APP_KEYCLOAK_CLIENT_ID=web_app
-
-2. By default, Docker Compose recreates the Keycloak container and resets the H2 database each time it is started. To persist Keycloak changes across restarts, you can make the following updates: 
-   * In your src/main/docker/keycloak.yml file, replace\
-   `
-   '-Dkeycloak.migration.strategy=OVERWRITE_EXISTING',` with 
-   ```
-   '-Dkeycloak.migration.strategy=IGNORE_EXISTING',
-   ```   
-   * In the same file, add a persistent volume under `volumes`:
+2. In this Blueprint generated project, Docker Compose persists Keycloak data across restarts by default. If you want your data to reset on restarts:
+   
+   * Edit the svc/keycloak.yml file. Replace '-Dkeycloak.migration.strategy=IGNORE_EXISTING',
+   with the following: 
+   ``` yaml
+   '-Dkeycloak.migration.strategy=OVERWRITE_EXISTING',
    ```
+ 
+   * In the same file, remove this persistent volume statement under `volumes`:
+   ``` yaml
    - ./keycloak-db:/opt/jboss/keycloak/standalone/data
    ```   
-   * Keycloak changes should now be persistent. You can reset your Keycloak database by emptying the `src/main/docker/keycloak-db` directory and restarting the container.
+   * Keycloak changes should now reset every time you restart.
 
-3. `User is not authenticated` Error message: If you see this message when you run the widgets, it is likely that your Keycloak application is not running. Check if the Docker Compose command is still running.
+2. `User is not authenticated` Error message: If you see this message while running the components, it is likely that your Keycloak application is not running or the .env.local is not configured properly. Check if the Docker Compose command is still running. Otherwise, check the parameters in microservice/conference-x/.env.local.
```

Comment:
"while running the components" -> "while components are running" (cleaner and takes the acting upon aspect out of the equation)

"the .env.local" -> "`.env.local`

"the Docker Compose command" -> can this be simplified to "Docker Compose"?

would backtick file in last sentence for consistency

Reply:
"while running the components" is awkward because components run without your oversight. you can tell the component to run and then it will be running, on its own. "you" have very little to do with the running component, so it makes sense to not introduce the user, even obliquely

Reply:
we are reading the process differently. Maybe it's the point of view. In this context, it is meant that while you are using the ent bundle run command, not while the component is running. You won't see see the message once the thing starts running. I've seen this usage before, running a program, running a command, running a component, here it is running a MFE. Thinking about how to make this sound better.